armi/reactor/blueprints/tests/test_gridBlueprints.py

```diff
@@ -242,7 +242,9 @@ def test_simple_read(self):
 
         gridDesign2 = self.grids["sfp"]
         _grid = gridDesign2.construct()
-        self.assertEqual(gridDesign2.gridContents[1, 1], "3")
+        self.assertEqual(gridDesign2.gridContents[1, 1], "1")
+        self.assertEqual(gridDesign2.gridContents[0, 0], "3")
+        self.assertEqual(gridDesign2.gridContents[-1, -1], "3")
```

Comment:
Can you add unit tests to cover the following scenarios:

- Cartesian, Full Core, Even
- Cartesian, Quarter Core, Odd
- Cartesian, Quarter Core, Even

The SFP test is nice, but let's make sure that we have the other variations tested too.

Reply:
Added quarter and even cases